include/avif/internal.h

```diff
@@ -146,7 +146,7 @@ void avifImageCopyNoAlloc(avifImage * dstImage, const avifImage * srcImage);
 // Copies the samples from srcImage to dstImage. dstImage must be allocated.
 // srcImage and dstImage must have the same width, height, and depth.
 // If the AVIF_PLANES_YUV bit is set in planes, then srcImage and dstImage must have the same yuvFormat.
-// Ignores the gainMap field (which exists only if AVIF_ENABLE_GAIN_MAP is defined).
+// Ignores the gainMap field.
```

Comment:
Should this be fixed? (Not sure if this is a to-do.)

Reply:
Ditto, not a todo.